napari/utils/context/_expressions.py

```diff
@@ -315,14 +315,21 @@ def __invert__(self) -> UnaryOp[T]:
         # note: we're using the invert operator `~` to mean "not ___"
         return UnaryOp(ast.Not(), self)
 
+    def __reduce_ex__(self, protocol: int) -> Tuple[Any, ...]:
```

Comment:
I think you may need:

```python
from typing import SupportsIndex
``` 

and 

```suggestion
    def __reduce_ex__(self, protocol: SupportsIndex) -> Tuple[Any, ...]:
```

Let's see how mypy complains with the current update first.

Reply:
Ok, also need to import Tuple, I'll do both at the same time.